buddyboss/buddyboss-platform
# Fixed - Text is overlapping under the play button

### All Submissions:


Fixes #2597 

### How to test the changes in this Pull Request:

1. Go to 'Forum reply'
2. add 5+ videos
3. See style

### Proof Screenshots or Video
https://ibb.co/sqTKXNg

### Other information:
